system/sensord/rawgps/rawgpsd.py

```diff
@@ -120,13 +122,37 @@ def setup_quectel(diag: ModemDiag):
 
   if gps_enabled():
     at_cmd("AT+QGPSEND")
+  #at_cmd("AT+QGPSDEL=0")
 
   # disable DPO power savings for more accuracy
   at_cmd("AT+QGPSCFG=\"dpoenable\",0")
   # don't automatically turn on GNSS on powerup
   at_cmd("AT+QGPSCFG=\"autogps\",0")
 
-  at_cmd("AT+QGPSSUPLURL=\"supl.google.com:7275\"")
+  # Do internet assistance
+  at_cmd("AT+QGPSXTRA=1")
```

Comment:
should we also remove the SUPL configuration? might make it difficult to debug with multiple assistance sources

Reply:
Yes that’s why I removed the url. I never got it to work anyway, and the other approach appears to be the standard.

Reply:
ah missed that it was deleted